airflow/serialization/serialized_objects.py

```diff
@@ -287,7 +287,7 @@ class BaseSerialization:
     _datetime_types = (datetime.datetime,)
 
     # Object types that are always excluded in serialization.
-    _excluded_types = (logging.Logger, Connection, type)
+    _excluded_types = (logging.Logger, Connection, type, property)
```

Comment:
What does this one affect?

Reply:
if we don't exclude it, the serialized object will have both `operator_extra_links` (added by the method `serialize_to_json` with the method as value) and `_operator_extra_links`, also the property type should be excluded anyway because it is not deserialized.